global-accessibility-awareness-day/2022.md

```diff
@@ -0,0 +1,35 @@
+# Global Accessibility Awareness Day 2022
+
+Thursday 19th May 2022
+
+## Testing your product with a screen reader
+
+In the UK around [2 million people live with sight loss][sight-loss]. There are 350,000 people on the registers of blind and partially sighted people in the UK.
+
+[sight-loss]: https://www.rnib.org.uk/professionals/knowledge-and-research-hub/key-information-and-statistics
+
+On Global Accessibility Awareness Day, we want to ask you to fire up a screen reader and spend half an hour browsing one of our products. It could be theguardian.com, one of our apps or a product your team maintains.
+
+If you'd like to give yourself a challenge, why not try our **Screen Reader Speedrun**:
+
+- open [theguardian.com](https://www.theguardian.com) in your favourite browser.
```

Comment:
Does VoiceOver perform ok on Chrome/Firefox? The WebAIM site suggests Safari might work best, but maybe it's actually fine? 🤷

Reply:
Good point. It _ought_ to work with Chrome / Firefox, but I have found the experience to be more frustrating with these browsers. Since the target audience for these steps includes people who may never have used a screen reader before, I think it's a good idea to set them on the Safari happy path, like WebAIM suggests 🙂

```suggestion
- on your laptop, open [theguardian.com](https://www.theguardian.com) in the Safari web browser.
```

Reply:
It's designed to work with Safari - I believe some useful features like the rotor only work on Safari.